extras/antennas/na-701.inp

```diff
@@ -0,0 +1,17 @@
+CM NA-701 Antenna
+CM
+CM
+CM
+CM
+CM forw: 0, 0 ; back: 0, 180
+CE
+GW 1 9 0 0 0 0 0 0.22 .0015
+GE 1
+LD 4 1 9 9 50
+GN 1
+EK
+EX 0 1 1 0 1 0 0
+FR 0 1 0 0 433 0
```

Comment:
It's been a while, but according to this the calculation starts @433MHz. But in the preset_information the frequency range starts @400MHz.

If that's the case I imagine there are issues with the calculation of signal strength as the resulting .aba file won't be defined for those ranges.

Best practice: Define your scope (e.g. 400 to 400*63*0.01 from the preset_information file) and add at least 5-10% to lower and upper limit.

Reply:
I changed this and re-ran the frequency sweep so it now covers 400mhz -> 600mhz. I think this should be enough?

Reply:
Looks ok now. As long as all signal calculations reasonable at lower and upper limits I'm good